media-gfx/iscan-plugin-gt-f720/iscan-plugin-gt-f720-0.0.1.2-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Authors
```

Comment:
What's this ebuild doing? If you're doing a version bump, don't do unnecessary changes to the old version.